src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -223,7 +223,7 @@ export default class TaskRoleContainerList extends React.Component {
         isResizable: true,
         onRender: (item) => {
           const gpuAttr = !isNil(item.containerGpus) && parseGpuAttr(item.containerGpus);
-          if (isNil(gpuAttr)) {
```

Comment:
please keep it unchanged, the `containerGpus: null` issue has been fixed in #3027

Reply:
Got it. Thanks.